marklogic-data-hub-central/ui/src/components/explore/graph-explore-side-panel/graph-explore-side-panel.tsx

```diff
@@ -139,7 +143,7 @@ const GraphExploreSidePanel: React.FC<Props> = (props) => {
     if (currentTab === DEFAULT_TAB) {
       return (
         <div aria-label="instance-view">
-          <TableView document={details} contentType="json" location={{}} isEntityInstance={true} isSidePanel={true} />
+          <TableView document={details} contentType="json" location={{}} isEntityInstance={true} isSidePanel={true} data={data.current} openUnmergeCompare={props.openUnmergeCompare} loadingCompare={props.loadingCompare} isUnmergeAvailable={props.isUnmergeAvailable}/>
```

Comment:
Please don't use a single line to have all the properties. It gets unreadable.
See this page as a guide for best practices: https://wiki.marklogic.com/display/ENGINEERING/Data+Hub+Central+React+Best+Practices

Reply:
Done!, Thanks for the info link!